content/en/apps/payments/send-payments.md

```diff
@@ -0,0 +1,261 @@
+        }
+    }
+    node {
+        name "O=PartyB,L=New York,C=US"
+        p2pPort 10008
+        rpcSettings {
+            address("localhost:10009")
+            adminAddress("localhost:10049")
+        }
+        cordapp project(":iou-workflows")
+        cordapp project(":iou-contracts")
+        rpcUsers = [[user: "user1", "password": "test", "permissions": ["ALL"]]]
+        cordapp("$corda_payments_release_group:payments-cordapp:$corda_payments_release_version") {
+            config """
+            {
+                "payments" : {
+                    "agent" : "O=Agent,L=New York,C=US"
+                }
+            }
+                """
+        }
+    }
+    node {
+        name "O=Agent,L=New York,C=US"
+        p2pPort 10000
+        rpcSettings {
+            address("localhost:10001")
+            adminAddress("localhost:10044")
+        }
+        cordapp("$corda_payments_release_group:payments-agent:$corda_payments_release_version") {
+            config """
+        {
+            "workflow" : {
+                "polling" : "1000",
+                "shutdown" : "true"
+                "timeout" : {
+                    "delay" : 50000
+                }
+            }
+        }
+            """
+        }
+        rpcUsers = [[user: "user1", "password": "test", "permissions": ["ALL"]]]
+    }
+}
+```
+
+4. From your project, run:
+
+  * `./gradlew clean deployNodes`
+
+  * `./build/nodes/runnodes`
+
+6. From directory where the downloaded `payments-spring-service` .jar files are located, run the following commands:
+
+```
+java -jar payments-spring-service-0.2.jar
+```
+
+You can now register payment accounts with the Payments Agent, using Modulr account details and use them to make payments against the Modulr PSP, from Corda.
+
+## Create a payment account
+
+You need to first create your payer and payee accounts on the Modulr sandbox. Once they have been created then you
+need to store the account details on the system. The details required for making a payment on Modulr from a GBP account
+to another GBP account are `Account ID` and `Account Number`.
+
+From the payer node and payee nodes you can add the account details as follows.
+
+Flow:
+````
+    val agent = paymentService.getPaymentAgent()
+    val accountName = "{your-account-alias}"
+    val currency = GBP
+    val psp = "MODULR"
+    val details = mutableMapOf(
+        "id" to "{your-account-id}",
+        "accountNumber" to "{your-account-number}"
+    )
+
+    paymentService.createAccount(accountName, currency, psp, details, agent)
+````
+RPC:
+```
+    rpc.startFlow(::CreatePaymentAccount, accountName, psp, details, agent)
+```
+
+When a payment account is registered on the system it returns a `PaymentAccount` object. The `PaymentAccount.accountId`
+is the parameter you will use to make a payment.
+
+## Make a payment
+
+Once the payer and payee have both registered their payment account details on the system then a payment can be made
```

Comment:
In reality you would register a payment account first, and then a beneficiary account to be your payment destination.

We can keep the current steps because they are simpler (which is perfect for quick start) but add the caveat that this is the path of least resistance. Again we don't want people to think you can only make payments to accounts you control or onboarded on corda payments.